src/app/globals.css

```diff
@@ -1,12 +1,34 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Waiting+for+the+Sunrise&display=swap');
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+:root {
+  /* Colors */
+  --color-primary: rgb(116 126 140);
+  --color-secondary: rgb(63 74 89);
+  --color-tertiary: rgb(31 42 63);
+  --color-quarternary: rgb(64 176 54);
+  --color-background: rgb(242 238 232);
+
+  /* Typography */
+  --font: 'Lato', sans-serif;
```

Comment:
Maybe be a little bit more specific about what you're using this font for. 
E.g. font-text for your general text or font-headline for the headlines.